src/app/stripe.service.ts

```diff
@@ -29,6 +29,9 @@ export class StripeService {
 
   private readonly fonts: CustomFontSource[];
 
+  private readonly secondaryColour = '#FF7272';
+  private readonly highlightColour = '#2AF135';
```

Comment:
I thought we had a list of all the colours in ts somewhere so we could dry this up but it looks like we don't. Maybe just add a comment to reference https://github.com/thebiggive/components/blob/c096dcaa499ece7080a363d4157bc2e1cd1c5c92/src/globals/variables.scss#L18 and https://github.com/thebiggive/components/blob/c096dcaa499ece7080a363d4157bc2e1cd1c5c92/src/globals/variables.scss#L19 for now?